config/src/main/java/org/springframework/security/config/annotation/web/configurers/ExpressionUrlAuthorizationConfigurer.java

```diff
@@ -127,7 +129,6 @@ private void interceptUrl(Iterable<? extends RequestMatcher> requestMatchers,
 	}
 
 	@Override
-	@SuppressWarnings("rawtypes")
```

Comment:
Thank you for the cleanups as well -- will you please place these in a separate commit so that each commit is about just one thing?